### PR TITLE
Update Production Configuration

### PR DESCRIPTION
**Pull request by sadiabbasi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "tonomy-app-websites",
   "private": true,
   "version": "0.0.0",
-  "type": "module",
```
Comment (theblockstalk) [inline package.json:5]: this is a major change. please investigate potential consequences. I've seen many things go wrong with this.

File: src/common/config/config.production.json [hunk 2]
```diff
@@ -10,16 +10,16 @@
         "mobileLogo": "/pangea/pangea-mobile-logo.png"
     },
     "links": {
-        "readMoreDownload": "https://pangea.web4.world.org",
-        "readMoreFoundation": "https://pangea.web4.world.org",
```
Comment (theblockstalk) [inline src/common/config/config.production.json:14]: this should be tonomy foundation website

File: src/common/config/config.production.json [hunk 3]
```diff
@@ -10,16 +10,16 @@
...
+        "readMoreDownload": "https://pangea.web4.world",
+        "readMoreFoundation": "https://pangea.web4.world",
         "playStoreDownload": "#",
```
Comment (theblockstalk) [inline src/common/config/config.production.json:15]: need to add these links

Comment (sadiabbasi) [reply src/common/config/config.production.json:15]: currently app is not published

File: package.json (hunk 1 above)
Comment (sadiabbasi) [reply package.json:5]: after adding ENABLE_EXPERIMENTAL_COREPACK environment variable getting this error https://vercel.com/tonomy-foundation/tonomy-app-websites/G26FFRBNHDDY4AGcmECBYLmh7GQ5 and i get this change from google

File: src/common/config/config.production.json [hunk 4]
```diff
@@ -10,16 +10,16 @@
         "mobileLogo": "/pangea/pangea-mobile-logo.png"
     },
     "links": {
-        "readMoreDownload": "https://pangea.web4.world.org",
-        "readMoreFoundation": "https://pangea.web4.world.org",
-        "playStoreDownload": "#",
-        "appleStoreDownload": "#"
+        "readMoreDownload": "https://tonomy.io",
```
[the remark this answers is not in the document]
Comment (sadiabbasi) [reply src/common/config/config.production.json:13]: get this from staging

**Pull request merged**